control/veda_val.py

```diff
@@ -91,7 +91,7 @@ def val_tokengen(self):
             'username': self.auth_dict['val_username'],
             'password': self.auth_dict['val_password'],
         }
-
+        print self.auth_dict
```

Comment:
No print statement should be used in code. If these are for debug purpose then better to use loggers.

Reply:
@muzaffaryousaf Sorry - will change, left it in by mistake